installer/client/src/pages/SystemCheckScreen.vue

```diff
@@ -0,0 +1,157 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <oxd-form :loading="isLoading">
+    <div class="orangehrm-system-check">
+      <oxd-text
+        tag="h5"
+        class="orangehrm-system-check-content orangehrm-system-check-title"
+        >System Check</oxd-text
+      >
+      <oxd-text class="orangehrm-system-check-content"
+        >In order for your orangeHRM installation to function properly,please
+        ensure that all the system check items listed below are green.if any are
+        red please take the necessary steps to fix them.</oxd-text
+      >
+      <div v-if="items">
+        <flex-table
+          v-for="item in items"
+          :key="item?.category"
+          :items="item?.checks"
+          :title-name="item?.category"
+        ></flex-table>
+      </div>
+      <oxd-form-actions class="orangehrm-system-check-action">
+        <oxd-button display-type="ghost" label="Back" @click="navigateUrl" />
+        <oxd-button
+          display-type="ghost"
+          label="Re-Check"
+          type="submit"
+          @click="Recheck"
+        />
+        <oxd-button
+          display-type="ghost"
+          :disabled="checkErrors"
```

Comment:
see my comment about computed properties above. `checkErrors` should be `hasErrors`